native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/ArgumentHandler.java

```diff
@@ -63,12 +70,15 @@
 public class ArgumentHandler {
     private final BMap<BString, Object> argumentsMap;
     private final MethodType method;
-
     private final BMap<BString, Object> fileInfo;
     private final BObject context;
     private final BObject field;
+    private final BObject responseGenerator;
+    private final Boolean validation;
+    private Boolean hasConstraintErrors;
```

Comment:
What's the purpose of defining this as a class variable?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1422/commits/e690ce44c5a3addeedce8feebb473c9785eba1f3